students/biegon_piotr/lesson_05_list_comprehensions/age_calculator.py

```diff
@@ -0,0 +1,20 @@
+avg = 0
+ages = []
+people = int(input("Enter the number of people: "))
+
+for x in range(people):
+    print("Person number", x + 1, "age: ")
```

Comment:
You can also do supply more parameters to `range` to start from specified number:
```
for x in range(1, people):
    print("Person number", x, "age: ")
```